### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by abhishektanwar**
Links in navbar has also been updated.

### CODE REVIEW

File: alerts.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+.success-alert{
+  color: #155724;
+  background-color: #d4edda;
+  border-color: #c3e6cb;
+}
+
+.secondary-alert{
+  color: #383d41;
+  background-color: #e2e3e5;
+  border-color: #d6d8db;
```
Comment (srishtigautam21) [inline alerts.css]: use var for colors

File: button.css [hunk 2]
```diff
@@ -5,4 +5,130 @@
   border: none;
   cursor: pointer;
   padding: 0;
+}
+
+.btn{
+  /* width: 60px; */
```
Comment (DivyanshSareen) [inline button.css]: remove comment, if not intended for future use

File: button.css [hunk 3]
```diff
@@ -5,4 +5,130 @@
...
+  font-size: 20px;
+  font-weight: 500;
+  border:0px;
+  border-radius: 10px;
+  cursor: pointer;
+}
+
+.btn-disabled{
+  /* cursor: not-allowed shows disabled sign */
+  /* cursor: not-allowed;  */
```
Comment (DivyanshSareen) [inline button.css]: remove comment

File: button.css [hunk 4]
```diff
@@ -5,4 +5,130 @@
...
+  color: white;
+}
+
+.btn-outline-primary{
+  border: 1px solid var(--primary-color);
+  color: var(--primary-color);
+  background-color: #fff;
+}
+.btn-outline-primary:hover{
+  /* border: 1px solid var(--primary-color); */
```
Comment (DivyanshSareen) [inline button.css]: remove comment

File: cards.css [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+.vertical-card{
+  max-width: 220px;
+  width: 100%;
+  position: relative;
+}
+
+.horizontal-card .image-container{
+  width: 120px;
+  height: 150px;
+}
```
Comment (DivyanshSareen) [inline cards.css]: use relative units

File: docsite.css [hunk 6]
```diff
@@ -1,3 +1,12 @@
+.nav-bar {
+	grid-area: header;
+	min-height: 10vh;
+	width: 100%;
+	z-index: var(--z-index-3);
+	/* position: sticky; */
```
Comment (DivyanshSareen) [inline docsite.css]: remove comment

File: modal.css [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0, 0, 0); /* Fallback color */
+  background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
+}
+
+/* .modal-content{
+	background-color: blue;
+	width: 100%;
+} */
```
Comment (DivyanshSareen) [inline modal.css]: commented code

File: ratings.css [hunk 8]
```diff
@@ -0,0 +1,40 @@
+.content {
+	width: 420px;
+	margin-top: 100px
+}
```
Comment (DivyanshSareen) [inline ratings.css]: try relative units here

**Review by DivyanshSareen — APPROVED** [review]
Great doc-site and components, just tidy up the code a little.

**Review by srishtigautam21 — APPROVED** [review]
good to merge

File: ratings.css (hunk 8 above)
Comment (abhishektanwar) [reply ratings.css]: fixed

File: docsite.css (hunk 6 above)
Comment (abhishektanwar) [reply docsite.css]: fixed

File: cards.css (hunk 5 above)
Comment (abhishektanwar) [reply cards.css]: fixed

File: modal.css (hunk 7 above)
Comment (abhishektanwar) [reply modal.css]: fixed

File: button.css (hunk 3 above)
Comment (abhishektanwar) [reply button.css]: fixed

File: button.css (hunk 2 above)
Comment (abhishektanwar) [reply button.css]: fixed

File: alerts.css (hunk 1 above)
Comment (abhishektanwar) [reply alerts.css]: fixed

**Pull request merged**